app/models.py

```diff
@@ -0,0 +1,46 @@
+from app import db
+
+
+class User(db.Model):   # User inherits from db.Model, a base class for all models from Flask-sqlalchemy
+    id = db.Column(db.Integer, primary_key=True)    # Fields are created as instances of the Column class, they take the field type as an argument
+    email = db.Column(db.String(120), index=True, unique=True)  # By indexing a value you can find it more easily in the db
+    first_name = db.Column(db.String(50), index=True, unique=False) # Do I need to say when something shouldn't be unique? Or just leave it?
+    last_name = db.Column(db.String(50), index=True, unique=False)
+    password_hash = db.Column(db.String(128))
+    masterclasses = db.relationship('MasterclassInstance', backref='instructor', lazy='dynamic')
+    booked_masterclasses = db.relationship('MasterclassAttendee', backref='attendee', lazy='dynamic')
+
+    def __repr__(self):
+        return '<User {} {}>'.format(self.first_name, self.last_name)
+
+
+class Masterclass(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(150))
+    description = db.Column(db.String(500))
+    timestamp = db.Column(db.DateTime, index=True)
+    masterclass_instances = db.relationship('MasterclassInstance', backref='location', lazy='dynamic')
```

Comment:
I'm a little nervous about having two "masterclass_instances" methods on different classes. Are there other things you could name these?